hal/src/thumbv6m/clock.rs

```diff
@@ -575,11 +575,7 @@ fn configure_and_enable_dfll48m(sysctrl: &mut SYSCTRL, use_external_crystal: boo
             // chill cycle disable
             w.ccdis().set_bit();
 
-            // usb correction is not set due to instability issues around
-            // USB bus resets. TODO(twitchyliquid64): Maybe switch to OSC8M?
```

Comment:
Oh I remember this. I was trying to get my USB bus driver to work, and found that everytime the host issued a USB reset then my chip locked up. Never really got to the bottom of it.

If you can make your system reset the USB device and it _doesnt_ lock up, then this comment is stale.